knowledge-base/how-to-use-odbc-connections-in-the-web-report-designer.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Using ODBC in the Web Report Designer
+description: Explains how to use ODBC connection in the Web Report Designer depending on the framework
+type: how-to
+page_title: How to work with ODBC connections in the Web Report Designer in .NET Core and .NET Framework applications
+slug: how-to-work-with-ODBC-connections-in-the-web-report-designer-in-new-core-and-new-framework-applications
+position: 
+tags: webreportdesigner,odbc
+ticketid: 1478987
+res_type: kb
+---
+
+## Environment
+<table>
+	<tbody>
+		<tr>
+			<td>Product</td>
+			<td>Progress® Telerik® Reporting</td>
+		</tr>
+    	<tr>
+			<td>Report designer</td>
+			<td>Web Report Designer</td>
+		</tr>
+	</tbody>
+</table>
+
+
+## Description
+ODBC connections are supported in the [Web Report Designer](./web-report-designer) as long as the System.Data.ODBC classes are supported in .NET Core 3.1 and up and out-of-the-box 
```

Comment:
Consider splitting into more sentences for clarity. For example:
"The ODBC connections are supported in the [Web Report Designer](./web-report-designer). For .NET Framework applications this comes out-of-the-box. In .NET Core 3.1+ they are supported as long as the System.Data.ODBC classes are supported."